### PR TITLE
add jobs resource

### PR DESCRIPTION
**Pull request by nofarpepperi**

### CODE REVIEW

File: entities/addons.ts [hunk 1]
```diff
@@ -118,3 +118,30 @@ export interface AddonFile extends AddonData {
...
+        header: any;
+        originalUrl: string;
+        body: any;
+        method: string;
+        query: any;
+    };
+    Status: string;
+    ExpirationDateTime: Date;
+    CreationDateTime: string | undefined;
+    ModificationDateTime: string | undefined;
```
Comment (Chasky-H) [inline entities/addons.ts:143]: no need fr these since your are extending AddonData

File: entities/addons.ts [hunk 2]
```diff
@@ -118,3 +118,30 @@ export interface AddonFile extends AddonData {
     PresignedURL?: string;
     FileVersion?: string;
 }
+
+export interface Job extends AddonData {
+    Key: string;
+    Version: string;
+    UserUUID: string;
+    NumberOfTry: number;
+    NumberOfTries: number;
```
Comment (Chasky-H) [inline entities/addons.ts:127]: Are there any optional fields?

**Review by Chasky-H — CHANGES_REQUESTED** [review]
Missing version bump

File: entities/addons.ts (hunk 1 above)
Comment (nofarpepperi) [reply entities/addons.ts]: fixed

File: entities/addons.ts (hunk 2 above)
Comment (nofarpepperi) [reply entities/addons.ts:127]: fixed

**Comment by nofarpepperi** [comment]
> Missing version bump

Oded changing the version also, waiting for his approval

**Pull request merged**